src/cunumeric/index/zip.h

```diff
@@ -45,6 +45,9 @@ class ZipTask : public CuNumericTask<ZipTask> {
 
 constexpr coord_t compute_idx(coord_t index, coord_t dim)
```

Comment:
the variable name `dim` is very confusing. I believe you meant `extent`, right? `dim` seems synonymous with `axis` in other contexts, so let's change this.